applications/stepswitch/src/stepswitch_inbound.erl

```diff
@@ -115,11 +115,22 @@ maybe_find_resource(_NumberProps, JObj) ->
 
 -spec maybe_add_resource_id(wh_json:object(), wh_proplist()) -> wh_json:object().
 maybe_add_resource_id(JObj, ResourceProps) ->
+    ResourceId = props:get_value('resource_id', ResourceProps),
     case props:get_is_true('global', ResourceProps) of
-        'false' -> JObj;
+        'false' ->
```

Comment:
@lazedo this seems like a copy paste error?  If the type is not a global resource the CCV Global-Resource is set to 'true'?  Also be very careful I think some older code assumes that if there is an authorizing-id then it was on-net as well as if there is a Resource ID then it is global....

I would prefer not modify the JObj if not global unless there is a strong need to do so.